src/sentry/analytics/events/first_replay_sent.py

```diff
@@ -8,6 +8,7 @@ class FirstReplaySentEvent(analytics.Event):
         analytics.Attribute("organization_id"),
         analytics.Attribute("project_id"),
         analytics.Attribute("platform", required=False),
+        analytics.Attribute("default_user_id", required=False),
```

Comment:
@JoshFerge It had to be literally user_id or it won’t work. Also, I added a new field called default_owner_id that doesn’t raise an exception

Reply:
i see on `FirstTransactionSentEvent` just has `default_user_id`?

Reply:
regardless, went ahead and updated

Reply:
@JoshFerge ok yes there’s a way to use a different ID field but you have to update the ETL repo and add the mapping and also we use the default owner as the user_id for almost every other backend event. Better for consistency’s sake: